src/commands/version.ts

```diff
@@ -22,43 +16,30 @@ export default class Version extends Command {
     }),
   }
 
-  async run(): Promise<VersionDetail> {
+  async run(): Promise<Partial<VersionDetail>> {
```

Comment:
Is this `Partial` needed? I see `VersionDetail` has `pluginVersions` as an optional prop.

Reply:
I'll try to remove it.